### ISSUE TITLE
Copyright field in the schema

### ISSUE THREAD

**Issue by drkarl** [fresh-standard/FRESCA#14]
Hi, Every time I want to update my CV I get blocked by the appeareance, since adding content always messes the formatting up in Word and when I want to renew the look I always spend a lot of time trying to find a Word template that I like and then the slow process to copy and rearrange all the information to the new template. So I had been thinking for a while that it would be great to decouple the data and the look and feel of the CV / résumé, then I saw HackMyResume in Hacker News, and I love it!! So congrats on the great job.

Now the actual issue. It is actually a feature request. It would be great to have an optional field for a copyright license (where you specify one of the many open source licenses or have a link to your own custom license).

The reason for this is that there are many unscrupulous recruiters that do a lot of bad practices on candidates' CVs. Some interesting articles on the subject:

http://www.stdlib.net/~colmmacc/2005/05/17/copyright-your-cv/

http://www.corcodilos.com/blog/317/let-the-resume-wars-begin

http://freelancing.stackexchange.com/questions/287/what-steps-can-i-take-to-protect-my-reputation-if-my-cv-is-plagiarised

Also: http://www.brandonsavage.net/why-recruiters-are-bad-for-your-career/

**Comment by hacksalot** [comment]
I see no reason why we couldn't. How formally this should be expressed in the schema, versus themes, versus folded into resume content, is another question.

Thanks for the links re: rascally recruiters. I've run into quite a bit of that myself.
